faf.spec.in

```diff
@@ -144,7 +144,7 @@ A plugin for %{name} handling ruby scripts problems.
 Summary: %{name}'s dnf plugin
 Requires: %{name} = %{faf_version}
 Requires: python3-dnf
-Obsoletes: %{name}-yum
+Obsoletes: %{name}-yum <= 1.3.5
```

Comment:
I think it should be < 1.3.5, since 1.3.4 is the last version ever.

Reply:
Corrected.